pkg/module/module_test.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 func TestManager_Register(t *testing.T) {
+	t.Skip("Test satisfied adequately by Linux tests")
```

Comment:
Please correct me if I'm wrong.

```suggestion
	if runtime.GOOS == "windows" {
		t.Skip("Test satisfied adequately by Linux tests")
	}
```

Reply:
fixed